bpf/Makefile

```diff
@@ -318,3 +318,12 @@ clean:
 		$(MAKE) $(SUBMAKEOPTS) -C $(TARGET) clean;)
 	$(QUIET)rm -fr *.o *.ll *.i *.s
 	$(QUIET)rm -f $(TARGET)
+
+
+BEAR_CLI   = $(shell which bear 2> /dev/null)
+gen_compile_commands:
```

Comment:
Any reason not to use the actual target (`compile_commands.json`) here with the correct dependencies (presumably all C code in the directory)?

Reply:
See comment in pr.